test/WebJobs.Script.Tests.Integration/Management/InstanceManagerTests.cs

```diff
@@ -438,7 +439,7 @@ public async Task ValidateContext_Succeeds_For_WebsiteUseZip_Only()
         {
             var environment = new Dictionary<string, string>()
             {
-                { EnvironmentSettingNames.AzureWebsiteZipDeployment, "http://microsoft.com" }
```

Comment:
Is this URL change needed? Testname says invalidzip url and microsoft.com is also invalid zip url right?

Reply:
Yes, it needs to be changed. This endpoint is now responding differently, breaking the tests. Additionally, tests should **not** be making HTTP requests like this at all - no external endpoint calls. They should all be mocked.